src/bundle/Resources/public/scss/_add-translation.scss

```diff
@@ -8,6 +8,7 @@
         flex-wrap: wrap;
         padding-bottom: calculateRem(140px);
 
+        .form-group:first-of-type .ibexa-label,
```

Comment:
After checking it, I believe that the problem should be fixed in the product-catalog, unless it will be hard to fix there.